docker/Dockerfile.psga-pipeline

```diff
@@ -42,14 +32,42 @@ WORKDIR /app/psga
 #    This error does not disappear if this is installed via conda
 RUN nextflow plugin install nf-amazon@1.11.0 \
   && pip install setuptools==63.1.0 \
-  && pip install --progress-bar=off --no-cache-dir --upgrade pip==${PIP_VERSION} \
-  && pip install --progress-bar=off --no-cache-dir poetry==${POETRY_VERSION} \
+  && pip install --progress-bar=off --no-cache-dir --upgrade pip \
+  && pip install --progress-bar=off --no-cache-dir poetry \
```

Comment:
Just to check, will this just install the latest version of `poetry` without telling it what version to install? If so, won't you end up in a situation where it installs a version of `poetry` that the `poetry.lock` file is incompatible with?

Reply:
This is true, hence the pinned numbers above. I wanted to find a more robust way of handling it, as there are conflicting version in the CI too, hence removed for now.